entity-framework/ef6/fundamentals/providers/index.md

```diff
@@ -36,6 +36,8 @@ Providers we are aware of that have been rebuilt for EF6 include:
     *   Npgsql is available as a [NuGet package](https://www.nuget.org/packages/EntityFramework6.Npgsql/)
 *   **Oracle**
     *   ODP.NET is available as a [NuGet package](https://www.nuget.org/packages/Oracle.ManagedDataAccess.EntityFramework/)
+*   **SQLite**
+    *   System.Data.SQLite is available as a [NuGet package](https://www.nuget.org/packages/System.Data.SQLite/)
```

Comment:
That is not a EF Core provider. are you thinking of this: https://www.nuget.org/packages/System.Data.SQLite.EF6 ?

Reply:
Actually, the **System.Data.SQLite** package includes support for EF6 since [version 1.0.92.0](https://www.nuget.org/packages/System.Data.SQLite/1.0.92.0) (March 19, 2014).

Excerpt from the [release notes](https://system.data.sqlite.org/index.html/doc/trunk/www/news.wiki):
> * Convert the primary NuGet package, "System.Data.SQLite", into a meta-package.
> * Enhancements to the NuGet packages, including the new "modular" packages.

I agree that it's rather counter intuitive that **System.Data.SQLite** depends on **System.Data.SQLite.EF6** and not the other way around. The ADO.NET provider is now in the **System.Data.SQLite.Core** package.